gem/spec/spec_helper.rb

```diff
@@ -13,7 +13,7 @@
 #  See the License for the specific language governing permissions and
 #  limitations under the License.
 
-require_relative '../lib/earlgrey'
+require 'earlgrey/cli'
```

Comment:
the spec should run using earlgrey from source code, not the earlgrey installed gem.

required_relative is how we ensure we're testing the actual earlgrey code.